### ISSUE TITLE
Taging a Release

### ISSUE THREAD

**Issue by phikal** [larsbrinkhoff/forth-mode#82]
For packages to be installable from MELPA Stable, a commit has to be tagged in the git repository. Therefore I would want to ask if it would be possible to tag a commit that could be considered "stable", for us MELPA Stable users?

**Comment by phikal** [comment]
Updating this request, I am now working on [NonGNU ELPA](https://elpa.nongnu.org), and would like to add this package to the archive. Compared to MELPA, it uses the last commit that bumped the version tag to create a new release, so all that has to be done is update the version tag in the package header, whenever a satisfying stable state is reached.

**Comment by larsbrinkhoff** [comment]
Sorry, I didn't see your message from two years back.

I can certainly make a tag.  As for the package header, do you mean this from forth-model.el?  `;; Version: 0.1`  I could bump that and also make a tag.

**Comment by phikal** [comment]
Yes, ELPA checks the "main file", in this case `forth-mode.el` for whatever is given after "Version", so it just has to be bumped to some greater value.

**Comment by larsbrinkhoff** [comment]
I bumped the package version and made tag too.

**Comment by phikal** [comment]
That will fix the MELPA Stable issue, but ELPA uses the master branch, and therefore won't find the commit. Would it be possible to append 38d5152011ee67e0cff9d4a5ddfb1f908e5be013 onto master?

**Comment by larsbrinkhoff** [comment]
That was the intent; just a minor git mishap.  Should be fixed now, no?

**Comment by phikal** [comment]
Yes, build the package can now be built. I will notify you when it has been added to NonGNU ELPA, so that you might update the documentation, if you are interested.

**Comment by larsbrinkhoff** [comment]
Thanks!
